library/Fission/Web/User.hs

```diff
@@ -27,21 +26,13 @@ type ResetRoute = "reset_password"
                   :> Auth.ExistingUser
                   :> Reset.API
 
-server
-  :: ( MonadLogger   m
-     , MonadDB       m
-     , MonadUnliftIO m
-     , MonadThrow    m
-     , MonadTime     m
-     , MonadReader                cfg m
-     , Has IPFS.Gateway           cfg
-     , Has AWS.DomainName         cfg
-     , Has AWS.AccessKey          cfg
-     , Has AWS.SecretKey          cfg
-     , Has AWS.ZoneID             cfg
-     , Has AWS.Route53MockEnabled cfg
-     )
+server ::
+  ( MonadDB       m
+  , MonadTime     m
+  , MonadDNSLink  m
+  , MonadLogger   m
+  )
   => ServerT API m
 server = Create.server
-    :<|> const Verify.server
+    :<|> (\_ -> Verify.server)
```

Comment:
these are the same thing, right? why prefer this?

Reply:
Yeah, they're the same. The lambda version is one character longer, and very clear to newcomers. Again, I'm not married to this, but thought I'd see if it made the logic easier to understand.

Thoughts?

Reply:
No real preference either way. `const` did confuse me when first getting into the code base, but now I'm used to it so don't even think twice.

I think it makes it slightly less readable but slightly easier to understand... if that makes any sense haha

Reply:
Yup totally makes sense! This is where I kind of get to — there are lots of things that more experiences Haskellers like, because they make sense as you become familiar with the patterns. Lenses are actually surprisingly readable/expressive _once you put the time in_, but there's that initial learning curve. I'm trying to figure out where the balance is 🤷‍♀